### PR TITLE
add more questions and answers to pun

### PR DESCRIPTION
**Pull request by muaz234**
add questions and the answers from line 269 to 296 as counted from my text editor. just for fun questions and answers

### CODE REVIEW

File: puns.json [hunk 1]
```diff
@@ -267,6 +267,33 @@
...
+    "punchline": "A: They both become useless when you open Windows"
+  },
+  {
+    "pun": "Q: What is a programmer's favourite hangout place?",
+    "punchline": "A: Foo Bar"
+  },
+  {
+    "pun": "Q: What is a programmer's favourite hangout place?",
+    "punchline": "A: Foo Bar"
+=======
```
Comment (AlexLakatos) [inline puns.json:296]: This is not valid JSON. Could you please fix it?

File: puns.json [hunk 2]
```diff
@@ -267,6 +267,33 @@
...
+    "pun": "Q: What do computers and air conditioners have in common?",
+    "punchline": "A: They both become useless when you open Windows"
+  },
+  {
+    "pun": "Q: What is a programmer's favourite hangout place?",
+    "punchline": "A: Foo Bar"
+  },
+  {
+    "pun": "Q: What is a programmer's favourite hangout place?",
+    "punchline": "A: Foo Bar"
```
Comment (AlexLakatos) [inline puns.json:295]: This looks like it's a duplicate pun.

File: puns.json [hunk 3]
```diff
@@ -267,6 +267,33 @@
...
+    "pun": "Q: What is the way of object oriented to become wealthy?",
+    "punchline": "A: Inheritance"
+  },
+  {
+    "pun": "Q: What is an algorithm?",
+    "punchline": "A: Word used by programmers when they did not want to explain what they did"
+  },
+  {
+    "pun": "Q: What do computers and air conditioners have in common?",
+    "punchline": "A: They both become useless when you open Windows"
```
Comment (AlexLakatos) [inline puns.json:287]: This pun already exists further up, in line 67.

File: puns.json [hunk 4]
```diff
@@ -267,6 +267,33 @@
     "punchline": "A: A*"
   },
   {
+    "pun": "Q: So, where did you two met?",
+    "punchline": "A: Github"
+  },
+  {
+    "pun": "Q: Who is your boyfriend?",
+    "punchline": "A: I'll leave the relations to the database"
+  },
```
Comment (AlexLakatos) [inline puns.json:276]: I didn't think these were particularly funny, could you remove them from the PR please?

**Review by AlexLakatos — CHANGES_REQUESTED** [review]
Thanks for the PR, I've got a few changes I wanted before I accept it.

**Comment by muaz234** [comment]
Hi Alex, i have made the necessary changes. Appreciate if you could review and merge them if everything is correct. 

Thanks

**Review by AlexLakatos — APPROVED** [review]
Thanks for making the changes! LGTM 💈

**Pull request merged**